src/uu/mv/src/mv.rs

```diff
@@ -70,13 +70,26 @@ static ARG_FILES: &str = "files";
 
 #[uucore::main]
 pub fn uumain(args: impl uucore::Args) -> UResult<()> {
-    let matches = uu_app()
-        .after_help(&*format!(
-            "{}\n{}",
-            LONG_HELP,
-            backup_control::BACKUP_CONTROL_LONG_HELP
-        ))
-        .get_matches_from(args);
+    let help = format!(
```

Comment:
it seems like an unrelated change, no ?

Reply:
That variable is created to avoid "temporary value dropped while borrowed" error:
```
  --> src/uu/mv/src/mv.rs:79:41
   |
79 |       let mut app = uu_app().after_help(&*format!(
   |  _________________________________________^
80 | |         "{}\n{}",
81 | |         LONG_HELP,
82 | |         backup_control::BACKUP_CONTROL_LONG_HELP
83 | |     ));
   | |     ^ - temporary value is freed at the end of this statement
   | |_____|
   |       creates a temporary which is freed while still in use
84 |       let matches = app
   |                     --- borrow later used here
   |
   = note: consider using a `let` binding to create a longer lived value
   = note: this error originates in the macro `format` (in Nightly builds, run with -Z macro-backtrace for more info)

For more information about this error, try `rustc --explain E0716`.
error: could not compile `uu_mv` due to previous error
```

Reply:
oh, right, thanks